src/components/Dashboard/TodoList.vue

```diff
@@ -816,7 +851,7 @@ export default {
       // open confirmation dialog and wait for response
       this.$refs.confirm.open(
         'Delete Draft?',
-        'Delete your ' + task.draftTitle + '? Any changes you\'ve made will be lost.',
+        'Delete your ' + task.title + '? Any changes you\'ve made will be lost.',
```

Comment:
This may break other correction types, as they were using "draftTitle". Check where the correction item is loaded to see what properties are set (draftTitle vs title).

Reply:
Will dig into this, i couldn't see the difference  other than 'Correction -' being present, which is why i opted for this. I had a feeling when you deleted a Draft correction previously, it wasn't specified that it was a Correction.

Reply:
These properties, `draftTitle` or `title`, as assigned when the Todo List item is loaded into memory. I believe different types of items have different properties. I think, here, drafts/correction may not have `draftTitle` and/or `title`. You can look at the code or test drafts + corrections to verify.

Reply:
After looking at both properties the naming is a little inconsistent for the `title` prop, will revert to draftTitle.